set-mongo/src/main/java/eu/europeana/set/mongo/service/PersistentUserSetServiceImpl.java

```diff
@@ -431,5 +431,28 @@ public void removeAll(List<PersistentUserSet> userSets) {
 	public PersistentUserSet update(PersistentUserSet userSet) throws  UserSetValidationException {
 		return store(userSet);
 	}
+	
+	/**
+	 * Getting the ids of the duplicate sets.
+	 */
+	public List<String> getDuplicateUserSetsIds(UserSet userSet) {
+	  
+	     if(userSet.getSubject()==null || userSet.getSubject().size()==0) return null;
+	       
+	     Query<PersistentUserSet> query = getUserSetDao().createQuery().disableValidation();
+	     query.filter(FIELD_TYPE, UserSetTypes.ENTITYBESTITEMSSET.toString());     
+	     query.filter(WebUserSetModelFields.SUBJECT + " size", userSet.getSubject().size());
```

Comment:
can we use the FilterOperator enum?

Reply:
I think we cannot, since the filter value that we need from the enum is private.

Reply:
The enumarations are always public see FilterOperator.NOT_EQUAL.value()

Reply:
I see that the enumeration I mentioned was part of morphia 1.3.2, in 2.x the operators are hardcoded ...